src/components/Landing/Landing.module.scss

```diff
@@ -202,3 +202,14 @@
   border-style: dashed;
   border-color: rgb(112 112 112);
 }
+
+@media screen and (max-width: 380px) {
```

Comment:
this'll work just fine, for future reference, I added a `xs` breakpoint at 375px so most of this can be done in tailwind CSS now.